vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -1385,14 +1385,16 @@ func resourceVSphereComputeClusterFlattenData(
 		d.Set("vsan_dit_rekey_interval", 0)
 	}
 
-	var dsIDs []string
-	if vsanConfig.DatastoreConfig != nil {
-		for _, ds := range vsanConfig.DatastoreConfig.(*vsantypes.VsanAdvancedDatastoreConfig).RemoteDatastores {
-			dsIDs = append(dsIDs, ds.Value)
+	if version.AtLeast(viapi.VSphereVersion{Product: version.Product, Major: 7, Minor: 1}) {
```

Comment:
is vsphere 7.0U1 considered 7.1? or 7.0.1. I believe you need to use the `Patch` field instead of `Minor` if that were the case?

Reply:
Nice catch - it's 7.0.1 and this would look for 7.1.

Reply:
```suggestion
	if version.AtLeast(viapi.VSphereVersion{Product: version.Product, Major: 7, Minor: 0, Patch: 1}) {
```

Reply:
Suggestion above.